### ISSUE TITLE
Naming conventions for REST setup of Drupal 8

### ISSUE THREAD

**Issue by arunz6161** [apigee/apigee-api-catalog-drupal#44]
When setting up [REST for API Catalog](https://www.drupal.org/docs/8/modules/apigee-api-catalog/expose-rest-apis-to-interact-with-api-docs), the Apigee entities, they are a little inconsistent. Naming conventions, basepaths, etc, should look more like `API Product`:

![Screenshot 2019-08-28 at 11 51 47 AM](https://user-images.githubusercontent.com/4381612/64282141-9ef4a800-cf09-11e9-9dac-d1dfc8d08662.png)

For example, to create an apidoc it uses POST /entity/apidoc, but to read the same resource we GET /api/{id}.

**Comment by arlina-espinoza** [comment]
@arunz6161 The user that reported this is using the core REST services module, while the [Apigee docs on drupal.org](https://www.drupal.org/docs/8/modules/apigee-api-catalog/expose-rest-apis-to-interact-with-api-docs) recommend using the JSON:API module/ecosystem. 
I don't think we need to support this use case, as we've already got documentation on setting everything up with JSON:API. For a longer comparison of the modules see this link:
https://www.drupal.org/docs/8/modules/jsonapi/jsonapi-vs-cores-rest-module
The TL:DR there pretty much explains why we chose this approach:

> Choose REST if you have non-entity data you want to expose. In all other cases, choose JSON:API. 

@arunz6161 LMK if we need to follow up on something, otherwise this can be closed.

**Comment by cnovak** [comment]
Closing, but can open up again if we get any comments back on this.
